apps/remix-ide/src/app/tabs/locales/zh/filePanel.json

```diff
@@ -9,6 +9,8 @@
   "filePanel.workspace.rename": "重命名工作空间",
   "filePanel.workspace.delete": "删除工作空间",
   "filePanel.workspace.deleteConfirm": "确定要删除当前工作空间?",
+  "filePanel.workspace.deleteAll": "Delete All Workspaces",
+  "filePanel.workspace.deleteAllConfirm": "Are you absolutely sure you want to delete all your workspace?",
```

Comment:
I think it should be `all your workspaces`

Reply:
Not sure if we should also add something like `Deleted workspaces can not be restored in any manner`

Reply:
thanks. will update